jwt/utils.py

```diff
@@ -1,17 +1,19 @@
 import base64
 import binascii
 import struct
+from typing import Union
 
 try:
+    from cryptography.hazmat.primitives.asymmetric.ec import EllipticCurve
     from cryptography.hazmat.primitives.asymmetric.utils import (
         decode_dss_signature,
         encode_dss_signature,
```

Comment:
Slightly off-topic, but there should probably be some kind of runtime check that the above functions were successfully imported (if only to provide better user feedback). See, for example; https://github.com/aio-libs/aiohttp-security/blob/master/aiohttp_security/session_identity.py#L21

Reply:
That already exists here. Please feel free to submit PRs to improve the user facing messaging.


https://github.com/jpadilla/pyjwt/blob/5bf8e70f26309b0a5ceda49ce0a7e97e89eee764/jwt/api_jws.py#L117-L120

Reply:
Right, I suppose these function are for internal use, so not relevant.